### PR TITLE
Basic Calculator Implementation

### PR DESCRIPTION
**Pull request by MuhammadMattiullah**

### CODE REVIEW

File: src/component/ContainerComponent.jsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+class ContainerComponent extends Component {
+
+    /* Constructor for state initialisation*/
+    constructor(props) {
+        super(props)
+        this.state = { celsiusTemp: '', fahrenheitTemp: '' }
+    }
+
+    /* Function For Calculating Fahrenheit Temperature */
+    convertCelsiusToFahrenheit = (value) => {
```
Comment (ifahadashraf) [inline src/component/ContainerComponent.jsx]: No brackets around single params

File: src/component/ContainerComponent.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        let fahrenheit = (value * (9 / 5) + 32);
+        return fahrenheit;
```
Comment (ifahadashraf) [inline src/component/ContainerComponent.jsx]: convertCelsiusToFahrenheit = value => (value * (9 / 5) + 32);

File: src/component/ContainerComponent.jsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+    /* Function For Calculating Celsius Temperature */
+    convertFahrenheiTtoCelsius = (value) => {
+        let celsius = (5 / 9) * (value - 32);
+        return celsius;
+    }
+
+    /* Function For Handling Celsius Temperature */
+    onChangeCelsius = (value) => {
+        if (value.length == 0 || isNaN(value)) {
```
Comment (ifahadashraf) [inline src/component/ContainerComponent.jsx]: if (!value.length || isNaN(value))

File: src/component/ContainerComponent.jsx [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+            this.setState({ fahrenheitTemp: value })
+            this.setState({ celsiusTemp: temp })
+        }
+    }
+
+    render() {
+        return (
+            <div style={{ textAlign: 'center' }}>
+                <InputComponent
+                    value={this.state.celsiusTemp}
```
Comment (ifahadashraf) [inline src/component/ContainerComponent.jsx]: value={ this.state.celsiusTemp }

File: src/component/ContainerComponent.jsx [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+                    scale="celsius"
+                    handleChange={this.onChangeCelsius}
+                />
+                <InputComponent
+                    value={this.state.fahrenheitTemp}
+                    scale="fahrenheit"
+                    handleChange={this.onChangeFahrenheit}
+                />
+                <p style={{ padding: '20px', fontWeight: 'bold' }}> The water can {this.state.celsiusTemp >= 100 ? "be boil" : "not be boil"} </p>
```
Comment (ifahadashraf) [inline src/component/ContainerComponent.jsx]:
```
<p ...>
  The water can
  {this.state.celsiusTemp >= 100 ? "be boil" : "not be boil"}
</p>
```

File: src/component/ContainerComponent.jsx (hunk 1 above)
Comment (MuhammadMattiullah) [reply src/component/ContainerComponent.jsx]: Updated!

File: src/component/ContainerComponent.jsx (hunk 2 above)
Comment (MuhammadMattiullah) [reply src/component/ContainerComponent.jsx]: Fixed!

File: src/component/ContainerComponent.jsx (hunk 5 above)
Comment (MuhammadMattiullah) [reply src/component/ContainerComponent.jsx]: Modified!

File: src/component/ContainerComponent.jsx (hunk 4 above)
Comment (MuhammadMattiullah) [reply src/component/ContainerComponent.jsx]: Spaced Added!

**Comment by ifahadashraf** [comment]
Very nice !

**Pull request merged**